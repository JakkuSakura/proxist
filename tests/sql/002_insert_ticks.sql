INSERT INTO ticks (tenant, shard_id, symbol, ts, seq) VALUES
('alpha', 'alpha-shard', 'AAPL', toDateTime64('2024-01-01 10:00:00', 6), 1),
('alpha', 'alpha-shard', 'MSFT', toDateTime64('2024-01-01 10:00:01', 6), 2);

SELECT tenant, symbol, count() AS rows
FROM ticks
GROUP BY tenant, symbol
ORDER BY tenant, symbol
FORMAT TSVWithNames;
