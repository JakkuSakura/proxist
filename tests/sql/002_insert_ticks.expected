tenant	symbol	rows
alpha	AAPL	1
alpha	MSFT	1
